connect/package.json

```diff
@@ -19,7 +19,7 @@
       "default": "./dist/*.js"
     }
   },
-  "devDependencies": {
+  "dependencies": {
```

Comment:
`typescript` and `de-cli` are dev dependencies for sure.

Reply:
Fixed.